scenes/menus/splash/splash.tscn

```diff
@@ -42,7 +42,7 @@ curve = SubResource("Curve2D_eqfq0")
23, 675.824, 1044.65, 675.504, 1044.06, 675.182, 1043.46, 674.857, 1042.87, 674.53, 1042.26, 674.201, 1041.66, 673.871, 1041.05, 673.538, 1040.44, 673.205, 1039.83, 672.869, 1039.22, 672.533, 1038.6, 672.196, 1037.98, 671.858, 1037.36, 671.519, 1036.74, 671.18, 1036.12, 670.84, 1035.5, 670.5, 1034.88, 670.16, 1034.26, 669.82, 1033.64, 669.481, 1033.02, 669.142, 1032.4, 668.804, 1031.78, 668.467, 1031.17, 668.131, 1030.56, 667.795, 1029.95, 667.462, 1029.34, 667.129, 1028.74, 666.799, 1028.13, 666.47, 1027.54, 666.143, 1026.94, 665.818, 1026.35, 665.496, 1025.77, 665.176, 1025.19, 664.859, 1024.62, 664.544, 1024.05, 664.233, 1023.48, 663.924, 1022.93, 663.619, 1022.37, 663.318, 1021.83, 663.02, 1021.29, 662.726, 1020.76, 662.436, 1020.24, 662.15, 1019.73, 661.869, 1019.22, 661.591, 1018.72, 661.319, 1018.23, 661.051, 1017.75, 660.789, 1017.28, 660.531, 1016.82, 660.279, 1016.37, 660.032, 1015.5, 659.556, 1014.67, 659.104, 1013.89, 658.676, 1013.16, 658.276, 1012.48, 657.903, 1011.85, 657.559, 1011.28, 657.246, 1010.76, 656.965, 1010.31, 656.717, 1009.6, 656.326, 1009, 656, 1008.3, 655.59, 1007.45, 655.095, 1006.92, 654.78, 1006.31, 654.422, 1005.63, 654.024, 1004.88, 653.585, 1004.07, 653.108, 1003.64, 652.855, 1003.2, 652.594, 1002.74, 652.323, 1002.26, 652.044, 1001.77, 651.757, 1001.27, 651.461, 1000.76, 651.157, 1000.23, 650.845, 999.685, 650.526, 999.13, 650.199, 998.563, 649.864, 997.984, 649.522, 997.393, 649.174, 996.791, 648.818, 996.178, 648.457, 995.555, 648.088, 994.921, 647.713, 994.277, 647.333, 993.624, 646.946, 992.961, 646.554, 992.289, 646.156, 991.609, 645.753, 990.92, 645.345, 990.223, 644.931, 989.518, 644.514, 988.806, 644.091, 988.087, 643.664, 987.361, 643.233, 986.629, 642.798, 985.891, 642.359, 985.147, 641.917, 984.398, 641.471, 983.643, 641.022, 982.884, 640.569, 982.121, 640.114, 981.353, 639.656, 980.582, 639.196, 979.807, 638.733, 979.029, 638.268, 978.248, 637.802, 977.465, 637.333, 976.68, 636.863, 975.893, 636.391, 975.104, 635.918, 974.314, 635.445, 973.524, 634.97, 972.733, 634.495, 971.942, 634.019, 971.151, 633.543, 970.36, 633.067, 969.571, 632.591, 968.782, 632.115, 967.995, 631.64, 967.21, 631.165, 966.427, 630.691, 965.646, 630.219, 964.868, 629.747, 964.094, 629.277, 963.322, 628.808, 962.555, 628.341, 961.791, 627.876, 961.032, 627.414, 960.278, 626.953, 959.529, 626.495, 958.785, 626.04, 958.047, 625.588, 957.316, 625.139, 956.59, 624.693, 955.871, 624.25, 955.16, 623.811, 954.455, 623.376, 953.759, 622.945, 953.07, 622.518, 952.39, 622.096, 951.718, 621.678, 951.056, 621.265, 950.403, 620.856, 949.759, 620.453, 949.126, 620.055, 948.502, 619.663, 947.89, 619.276, 947.288, 618.895, 946.698, 618.521, 946.12, 618.152, 945.553, 617.79, 944.999, 617.435, 944.457, 617.086, 943.928, 616.744, 943.413, 616.41, 942.911, 616.082, 942.423, 615.763, 941.95, 615.451, 941.49, 615.147, 941.046, 614.851, 940.618, 614.563, 940.204, 614.284, 939.426, 613.752, 938.713, 613.256, 938.07, 612.798, 937.498, 612.379, 937, 612)
```

Comment:
Don't think you meant to commit this. I should change this tool script to do less when the scene loads!

Reply:
Oops sorry, will revert it!